vault/login_mfa.go

```diff
@@ -1442,6 +1442,7 @@ func (b *MFABackend) mfaConfigToMap(mConfig *mfa.Config) (map[string]interface{}
 		}
 		respData["mount_accessor"] = mConfig.MountAccessor
 		respData["username_format"] = mConfig.UsernameFormat
+		respData["use_passcode"] = oktaConfig.UsePasscode
```

Comment:
Mirrors how the duo MFA works:
https://github.com/hashicorp/vault/blob/58a5b77bc6f0a5ac47e3e3d6020148b585f7213a/vault/login_mfa.go#L1452